### PR TITLE
[doc] Added and updated description of useful test applications

### PR DESCRIPTION
**Pull request by ethouris**
`srt-multiplex.md` - changed to `srt-test-multiplex.md` and updated

Added `srt-test-relay.md`.

### CODE REVIEW

**Review by maxsharabayko — COMMENTED** [review]
The "sample applications" section of the [`docs/README.md`](https://github.com/Haivision/srt/tree/master/docs#sample-applications) also needs an update then.

**Comment by ethouris** [comment]
Ok, updated, and I also added one more lacking description :)

File: docs/README.md [hunk 1]
```diff
@@ -46,12 +46,20 @@
...
-| [Using the<br /> `srt-tunnel` App](apps/srt-tunnel.md)       |         [apps](apps/)         | [srt-tunnel.md](apps/srt-tunnel.md)               | A sample application to set up an SRT tunnel for TCP traffic. |
-| <img width=200px height=1px/>                                | <img width=100px height=1px/> | <img width=200px height=1px/>                     | <img width=500px height=1px/>                                |
+| [Using the<br /> `srt-file-transmit` App](apps/srt-file-transmit.md) | [apps](apps/)         | [srt-file-transmit.md](apps/srt-file-transmit.md) | A sample application to transmit a file over SRT              |
+| [Using the<br /> `srt-tunnel` App](apps/srt-tunnel.md)               | [apps](apps/)         | [srt-tunnel.md](apps/srt-tunnel.md)               | A sample application to set up an SRT tunnel for TCP traffic. |
+| <img width=200px height=1px/>                                | <img width=100px height=1px/> | <img width=200px height=1px/>                     | <img width=500px height=1px/>                                 |
+
+## Testing applications
```
Comment (mbakholdina) [inline docs/README.md:56]: There is no need to create a separate table for these 2 items. Please move everything under "Sample Applications" table and delete the current one.

Comment (mbakholdina) [reply docs/README.md:56]: Please also change the folder to "apps" with an appropriate link.

File: docs/apps/srt-test-multiplex.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
-## srt-multiplex
+## srt-test-multiplex
```
Comment (mbakholdina) [inline docs/apps/srt-test-multiplex.md:1]: Something is wrong with the level of headings in this document. It should start with "# srt-test-multiplex" and then "## Usage", etc.. Could you please go through the document and update the levels carefully?

File: docs/apps/srt-test-relay.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## srt-test-relay
```
Comment (mbakholdina) [inline docs/apps/srt-test-relay.md:1]: Again, the level of heading is incorrect. Please change ## to #.

Comment (mbakholdina) [reply docs/apps/srt-test-relay.md:1]: I've also noticed that in srt-tunnel document the headings formatting is different. Could you please update it to look similar? Plus please change the title from "SRT Tunnel" to "srt-tunnel" for consistency. Thanks!

**Review by mbakholdina — CHANGES_REQUESTED** [review]
Hi @ethouris, thank you for updating the documentation! A couple of minor comments from me.

**Comment by ethouris** [comment]
Ok, I added fixes, please check again if everything's ok

File: docs/README.md [hunk 4]
```diff
@@ -46,12 +46,14 @@
...
+| :--------------------------------------------------------------------- | :-------------------- | :-------------------------------------------------- | :------------------------------------------------------------ |
+| [Using the<br /> `srt-live-transmit` App](apps/srt-live-transmit.md)   | [apps](apps/)         | [srt-live-transmit.md](apps/srt-live-transmit.md)   | A sample application to transmit a live stream from<br />source medium (UDP/SRT/`stdin`) to the target medium<br />(UDP/SRT/`stdout`). |
+| [Using the<br /> `srt-file-transmit` App](apps/srt-file-transmit.md)   | [apps](apps/)         | [srt-file-transmit.md](apps/srt-file-transmit.md)   | A sample application to transmit a file over SRT              |
+| [Using the<br /> `srt-tunnel` App](apps/srt-tunnel.md)                 | [apps](apps/)         | [srt-tunnel.md](apps/srt-tunnel.md)                 | A sample application to set up an SRT tunnel for TCP traffic. |
+| [Using the<br /> `srt-test-multiplex` App](apps/srt-test-multiplex.md) | [testing](testing/)   | [srt-test-multiplex.md](apps/srt-test-multiplex.md) | Testing app that allows to send multiple streams over one UDP link |
```
Comment (mbakholdina) [inline docs/README.md:54]: Folder -> `[apps](apps/)` for the last 2 rows of the table.

Comment (mbakholdina) [reply docs/README.md:54]: Please add the dots at the end of description sentence for `srt-file-transmit` and the last 2 applications.

File: docs/apps/srt-tunnel.md [hunk 5]
```diff
@@ -1,8 +1,7 @@
-SRT Tunnel
-==========
+# SRT Tunnel
```
Comment (mbakholdina) [inline docs/apps/srt-tunnel.md:1]: Change to `srt-tunnel` for consistency please.

File: docs/README.md (hunk 4 above)
Comment (mbakholdina) [reply docs/README.md:54]: I would also suggest changing "app" to "application" in the description for the last 2 rows of the table.

**Review by mbakholdina — CHANGES_REQUESTED** [review]
@ethouris , a couple of suggestions.

**Review by mbakholdina — APPROVED** [review]

**Pull request merged**